drupal8-example/Makefile

```diff
@@ -0,0 +1,22 @@
+up:
+ifeq ($(shell uname), Darwin)
+	@echo "Looks like you're on macOS! Using docker-compose-mac.yml"
+	@docker-compose -f docker-compose.yml -f docker-compose-mac.yml up -d
```

Comment:
Why hide what command is actually run?

Transparency into what magic is actually happening behind the scenes seems like a good idea (and could even remove the need for the echo above).

Reply:
Good point. I'll fix it.